APP/app.test.js

```diff
@@ -101,15 +101,58 @@ describe("GET /api/articles", () => {
         });
       });
   });
-  it("Articles should be sorted by date in descending order", () => {
+  it("Articles should be sorted by date in descending order by default", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({ body }) => {
         expect(body.articles).toBeSortedBy("created_at", { descending: true });
       });
   });
-});
+
+  it("Should respond with articles filtered by topic when queried", () => {
+    return request(app)
+    .get("/api/articles?topic=cats")
+    .expect(200)
+    .then(({body}) => {
+        body.articles.forEach((article) => {
+            expect(article.topic).toEqual("cats");
+    })
+    })
+  })
+  it("Should return a 400 error when query is invalid", () => {
+    return request(app)
+    .get("/api/articles?topic=othertopic")
+    .expect(400)
+    .then(({body}) => {
+        expect(body.msg).toEqual("Bad request")
+    })
+  })
+
+  it("Should respond with articles sorted by any valid column, defaults to date", () => {
+    return request(app)
+    .get("/api/articles?sort_by=votes")
+    .expect(200)
+    .then(({body}) => {
+    expect(body.articles).toBeSortedBy("votes", { descending: true })
+    })
+  })
+
+
+it("Should return a 400 error when query is invalid", () => {
+    return request(app)
+    .get("/api/articles?sort_by=badrequest")
+    .expect(400)
+    .then(({body}) => {
+        expect(body.msg).toEqual("Bad request")
+    })
+  })
+
+})
```

Comment:
Great testing so far! I'd add a few more tests for:

Status 200, accepts order query, e.g. **?order=desc**
Status 200, a valid topic query, but has no articles responds with an empty array of articles, e.g. **?topic=paper.**
Status 400. invalid order query, e.g. **?order=bananas**

Please organise your tests and keep your 200s together 👍 this makes your code look cleaner!